test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -240,19 +240,27 @@ private Map<String, Object> getTier2Env() {
         return parseSystemPropertyAsMap("tier2Env");
     }
 
-    private Map<String, Object> parseSystemPropertyAsMap(String systemProperty) {
+    public static Map<String, Object> parseSystemPropertyAsMap(String systemProperty) {
         String value = System.getProperty(systemProperty);
         checkNotNullOrEmpty(value, systemProperty);
-        log.info("Parsing {} = {}", systemProperty, value);
-        Map<String, String> split = Splitter.on(',').trimResults().withKeyValueSeparator("=").split(value);
-        return split.entrySet().stream().collect(Collectors.toMap(Map.Entry::getKey, e -> {
-            try {
-                return Integer.parseInt(e.getValue());
-            } catch (NumberFormatException ex) {
-                // return all non integer configuration as String.
-                return e.getValue();
-            }
-        }));
+        return parseSystemPropertyAsMap(systemProperty, value);
+    }
+
+    @NonNull
+    public static Map<String, Object> parseSystemPropertyAsMap(String systemProperty, String value) {
```

Comment:
This is completely changing the behavior of this method, right? Previously, it was returning a map in which all values were `Integer`, and now it is putting some string in the value. How is this impacting the configuration of system tests? Are system tests working and configured as expected with this change?